pkg/cliutil/prepare/prepare.go

```diff
@@ -231,13 +232,30 @@ func CheckClusterPortConflict(clusterSpec *spec.SpecManager, clusterName string,
 		}
 
 		metadata.GetTopology().IterInstance(func(inst spec.Instance) {
+			uniqueHosts := set.NewStringSet()
```

Comment:
Do you have tested this PR? I think the `uniqueHosts := set.NewStringSet()` should be placed out of `IterInstance`.

Reply:
It should be in the loop, it "unique" for on cluster, not for all clusters

Reply:
Yes, it should be placed in the `for` loop, but above the L234.